api/src/main/java/org/pmiops/workbench/api/RuntimeController.java

```diff
@@ -272,6 +273,36 @@ private Runtime getOverrideFromListRuntimes(String workspaceNamespace) {
     return ResponseEntity.ok(new EmptyResponse());
   }
 
+  @Override
+  public ResponseEntity<EmptyResponse> updateRuntime(
+      String workspaceNamespace, UpdateRuntimeRequest runtimeRequest) {
+    if (runtimeRequest == null || runtimeRequest.getRuntime() == null) {
+      throw new BadRequestException("Runtime cannot be empty for an update request");
+    }
+
+    if (runtimeRequest.getRuntime().getGceConfig() == null
```

Comment:
opt: could combine these into a singular xor statement

Reply:
Going to leave them separate because I have different messages for some of the cases.